packages/tc-ui/src/primitives/boolean/server.jsx

```diff
@@ -17,7 +17,7 @@ const Checkbox = ({ name, checkboxValue, disabled, userValue }) => {
 				value={checkboxValue.toString()}
 				aria-label={label}
 				id={`radio-${name}-${label}`}
-				checked={userValue === checkboxValue ? 'true' : null}
+				defaultChecked={userValue === checkboxValue ? 'true' : null}
```

Comment:
Similar to the select defaultChecked only needing us to pass in the actual user value, ios this the correct thing to do here? I'm not sure
```suggestion
				defaultChecked={userValue}
```

Reply:
I think this will not work - if you pick 'No' - when you go back to edit button it will not select it as `userValue = false` and if you picked 'Yes` in the edit page it will pick 'No' as your selection as it is the second (both Yes and No have value of defaultSelected = true)

Reply:
ok, I think I follow. Do we have cypress tests for his scenario?